.travis.yml

```diff
@@ -10,6 +10,8 @@ matrix:
   - jdk: openjdk7
   - jdk: openjdk8
   - jdk: oraclejdk8
```

Comment:
:question: Does it make sense to add 9 and 10 at the same time..?

Reply:
No, only 8 & 11 are long term support. See https://github.com/diffblue/deeptest-utils/pull/68/files#r302478359